src/Provider/Provider.Web/Views/ApplicationReviews/ApplicationReviewsToShareConfirmation.cshtml

```diff
@@ -1,19 +1,84 @@
 ﻿@using System.Net
+@using Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore.Projections.VacancyApplications
 @model Esfa.Recruit.Provider.Web.ViewModels.ApplicationReviews.ShareMultipleApplicationReviewsConfirmationViewModel
 
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
 
-          <section class="das-highlight">
-            <p class="govuk-heading-m">Share Multiple Applications</p>
-          </section>
-
-        <div class="govuk-body">
-            <p>
-                TODO: You have selected @Model.ApplicationReviewsToShare.Count() applications to share!
-            </p>
+        <h1 class="govuk-heading-l govuk-!-margin-bottom-6">
+            Share Multiple Applications
```

Comment:
Minor, but "Multiple Applications" is not in caps in the screenshot on the ticket